components/common/ImageUploader/index.tsx

```diff
@@ -13,9 +13,17 @@ interface ImageUploaderProps {
   value?: string | null;
   onChange: (value: string | null) => void;
   className?: string;
+  emptyIcon?: React.FC<React.SVGProps<SVGSVGElement>>;
 }
 
-const ImageUploader: FC<ImageUploaderProps> = ({ width = 104, height = 104, onChange, value, className }) => {
+const ImageUploader: FC<ImageUploaderProps> = ({
+  width = 104,
+  height = 104,
+  onChange,
+  value,
+  className,
+  emptyIcon: EmptyIcon = IconImage,
```

Comment:
컴포넌트 생성 함수보다 컴포넌트를 받는게 더 낫지 않을까?
```suggestion
  emptyIcon = <IconImage />
```